clang/test/Analysis/Checkers/WebKit/unretained-call-args.mm

```diff
@@ -329,13 +329,17 @@ void foo() {
   }
 }
 
+#define YES 1
```

Comment:
Is it just @YES/@NO that are safe constants? What's the expected behavior for say `#define RANDOM 3` and  `boo(@ RANDOM);`?

Reply:
Those are also considered safe. Basically @ any number should be treated as safe.